packages/exchange/src/pods/order/order.controller.ts

```diff
@@ -84,7 +84,7 @@ export class OrderController {
     }
 
     @Get()
-    @UseGuards(AuthGuard(), ActiveUserGuard)
+    @UseGuards(AuthGuard())
```

Comment:
Why?

Reply:
According to https://energyweb.atlassian.net/wiki/spaces/OR/pages/1012989957/Organizations+and+users all users have access to Exchange where orders are listed

Reply:
/orders endpoint is for managing organization orders. Non active user should not be able to use it.